python/nano/src/bigdl/nano/deps/ipex/ipex_inference_model.py

```diff
@@ -81,6 +83,13 @@ def __init__(self, model: torch.nn.Module, input_sample=None, use_ipex=False, dt
                                                         precision="fp32",
                                                         thread_num=thread_num)
         self.thread_num = thread_num
+        if (self.use_ipex is True or self.use_jit is True) and inplace is False:
+            # patch original model's attr to current new model
+            for attr in dir(self.original_model):
+                if attr not in dir(self) and not attr.startswith('_') and not\
+                        isinstance(getattr(self.original_model, attr), torch.nn.Module):
+                    setattr(self, attr, getattr(self.original_model, attr))
+            del self.original_model  # save memory
```

Comment:
This may not save memory.

If we write:
```python
class A():
    def m(self):
        ...
a = A()
b = getattr(a, "m")
del a
```

Then b will contains object `a` implicitly, because the first parameter of `m` is `self`. `del a` will only delete a reference to object `a`.

Reply:
Actually, this way has almost the same memory consumption as before, and after our discussion, `self.original_model = model` is just a reference, don't cause too much additional overhead.

I think current implementation (monkey patch) might be an acceptable choice, as if just use `self.original_model = model`, quantization will cause error as some behavior of `nn.Module`.

@MeouSker77 @TheaperDeng